tools/ansible/roles/dlpx.appliance-build-bootstrap/tasks/install_zfsutils.yml

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2018 by Delphix. All rights reserved.
+#
+
+- name: Install zfsutils | Install dependencies
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - zfsutils-linux
```

Comment:
I think we also depend on the ZFS kernel modules (`zfs-dkms`?). This dependency is currently implicit, but it would be great to make it explicit in this Ansible role. Of course, this is a pre-existing issue, so I'm fine if you don't want to deal with it in this change.

Reply:
the kernel module comes by default with the ubuntu kernel package.. thus, we'd have to explicitly depend on the kernel package, which seems like that's a little unnecessary? maybe just a comment is good enough? I could go either way.